libr/cons/stiv.c

```diff
@@ -53,9 +53,12 @@ static void render_ansi(PrintfCallback cb_printf, const ut8 *c, const ut8 *d) {
 }
 
 static int rgb(int r, int g, int b) {
-	if (r<0) r=0; if (r>255) r = 255;
-	if (g<0) g=0; if (g>255) g = 255;
-	if (b<0) b=0; if (b>255) b = 255;
+	if (r<0) r=0;
```

Comment:
in r2land we use braces even for single statements, but this is clearly a DIM to constraint the value between 0 and 255, so we can use `r = R_DIM (r, 0, 255);` instead of doing this (this was written like this because the original code was living outside r2land and didnt had that macro)

Reply:
Made the recommended change in a new commit in this branch.